export default function handleError(error) {
  throw error
}
